hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/NearCacheIsNotSharedTest.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Can you move this to `com.hazelcast.client.map.impl.nearcache` ?

Reply:
done